arionBackend/tests/serializers/test_json.py

```diff
@@ -0,0 +1,34 @@
+"""
+This module holds the test cases for the json serializer.
+"""
+from django.test import TestCase
+
+from arionBackend.models.hierarchy import Hierarchy
+from arionBackend.serializers.json import serialize_hierarchy_overview, serialize_hierarchy_complete
+
+
+class JsonTestCase(TestCase):
+    """
+    This class contains the tests for the Hierarchy API.
+    """
+
+    @classmethod
+    def setUpTestData(cls):

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method "setUpTestData" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS100)